Automattic/vip-go-mu-plugins
# Adds admin notice plugin

#  Description

First draft of the Notice plugin, that will enable us to add notices to wp-admin page

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

Update the code and add a new notice.

Check the admin page and see the new notice.
